WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -822,9 +822,21 @@ def processJobOutputs(self):
       if not outputSE and not self.defaultFailoverSE:
         return S_ERROR('No output SEs defined in VO configuration')
 
-      result = self.__transferOutputDataFiles(outputData, outputSE, outputPath)
-      if not result['OK']:
-        return result
+      result_transferODF = self.__transferOutputDataFiles(outputData, outputSE, outputPath)
+
+      # now that we (tried to) transfer the output files,
+      # including possibly oversized Output Sandboxes,
+      # we delete the local output sandbox tarfile in case it's still there.
+      if not result_sbUpload['OK']:
+        outputSandboxData = result_sbUpload.get('SandboxFileName')
+        if outputSandboxData:
+          try:
+            os.unlink(outputSandboxData)
```

Comment:
Maybe some log message?

Reply:
No need, really, this is just cleaning out a tmp file.